synfig-studio/src/gui/dialogs/dialog_setup.cpp

```diff
@@ -1446,6 +1461,18 @@ Dialog_Setup::on_brush_path_remove_clicked()
 	//! TODO if list size == 0: push warning to warning zone
 }
 
+void
+Dialog_Setup::on_plugin_path_change_clicked()
+{
+	String foldername = ResourceHelper::get_plugin_path();
+	//processes the path from get_plugin_path()
+	size_t pos = foldername.find('/');
+    while (pos != std::string::npos) {
+        foldername.replace(pos, 1, "\\");
+        pos = foldername.find('/', pos + 2);
+    }
```

Comment:
Is this really needed on Windows? (It won't work on Linux or MacOS by the way)

Besides, there are indentation issues.

Reply:
So these slash-to-backslash replacement is really needed, right?

Reply:
no this isnt needed i tried on windows and it works without it. Im asking if theres some way to add it for mac linux